bindings/web5_uniffi_wrapper/src/dids/bearer_did.rs

```diff
@@ -21,10 +24,13 @@ pub struct BearerDidData {
 pub struct BearerDid(pub InnerBearerDid);
 
 impl BearerDid {
-    pub fn new(uri: &str, key_manager: Arc<dyn KeyManager>) -> Result<Self> {
+    pub fn new(did: Did, document: Document, key_manager: Arc<dyn KeyManager>) -> Self {
```

Comment:
eh? we can't do did.document?

Reply:
oh right nm, this is the query / params / uri part

Reply:
then what benefit do we have for this vs just the string uri? Sometimes the string is more convenient to the user I would guess

Reply:
I agree actually. This is an area where I diverged from the old SDKs so I was moving back in line with them. But I think there is room for both: instantiation from the constituent parts, but also from a DID URI (which encapsulates resolution). If you agree, stub in a ticket on the backlog!